lib/translator/translation.cc

```diff
@@ -159,6 +162,10 @@ ApiStatus Translation::Complete(Span<const nvme::GenericQueueEntryCpl> cpl_data,
       pipeline_status_ = ReadToScsi(buffer, nvme_cmds_[0], kLbaSize);
       ret = ApiStatus::kSuccess;
       break;
+    case scsi::OpCode::kSync10:
```

Comment:
maybe put a todo if we need to handle something in the gen translator

Reply:
all the necessary translations should be handled by this https://github.com/googleinterns/scsi2nvme/pull/150 so once that is merged then we're all good.